lib/head.js

```diff
@@ -81,7 +85,7 @@ function unique () {
           } else {
             const category = h.props[metatype]
             const categories = metaCategories[metatype] || new Set()
-            if (categories.has(category)) return false
+            if (categories.has(category) && ALLOWED_DUPLICATES.indexOf(category) < 0) return false
```

Comment:
`!== -1` is the correct way to check this 🕵️

Reply:
IMO you are wrong. If category is not found (```<0``` or ```===-1```) within ALLOWED_DUPLICATES we prevent duplicate otherwise we allow it 😉

Reply:
I agree with @timneutkens 

> Return value: The first index of the element in the array; **-1 if not found.**

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf#Return_value

Reply:
Hi @timneutkens @stramel 

thanks for the answers. Ok, let's clarify this because I think we have misunderstandings here.
So line 88:

https://github.com/davscro/next.js/blob/72a595556bffb17b16f5680ba7be45e88e03873f/lib/head.js#L88

has a purpose to prevent adding duplicates (```returns false```), right?
So the first part ```categories.has(category)``` was there already which means if we have the category we immediately return false to prevent duplication, ok? So to allow duplication we have ```ALLOWED_DUPLICATES``` const array, which means if we **don't find** a category within the array (```=== -1```) the statement became ```TRUE``` and we also immediately return false.
So line became 
```if(TRUE && TRUE) return false``` right?

And if we have the tag in const array
```if(TRUE && FALSE (index will be >=0)).....}else{ will be fired with adding duplication }``` right?

So to conclude, in the first commit I had:
```!ALLOWED_DUPLICATES.includes(category)``` - please note exclamation mark
translated to indexOf should be:
```!ALLOWED_DUPLICATES.indexOf(category) !== -1``` - please note exclamation mark
so my final change was (to avoid negation):
```ALLOWED_DUPLICATES.indexOf(category) < 0```- no negation

So @timneutkens , my question is if you want to have ```!== -1``` (what you proposed) then we should negate it as before otherwise we could keep either ```<0``` or ```===0```, what do you think?

Reply:
This is the reason I asked to add tests 😄 I was sleeping when I wrote that comment and you are right 👍 The reason I didn't want to use `<0` because it's confusing when you come back to it later 😄